src/content/release/breaking-changes/snackbar-with-action-behavior-update.md

```diff
@@ -0,0 +1,92 @@
+---
+title: SnackBar with action no longer auto-dismisses
+description: >-
+  SnackBars with an action button now default to not auto-dismissing unless
+  manually dismissed by the user.
+---
+
+## Summary
+
+The default behavior of a [`SnackBar`][] with an action has changed. Previously, a
+`SnackBar` with an action would only not auto-dismiss if talkback was enabled. 
```

Comment:
```suggestion
`SnackBar` with an action would not auto-dismiss if talkback was enabled. 
```

The "only" added confusion, for me, at least. Too many negatives, in effect. But, does this make the text wrong?

Reply:
Removing it still sounds correct to me. Thank you for the improvement!